### PR TITLE
style: correct text direction.

### PR DESCRIPTION
**Pull request by ceoaliongroo**
screen shot 2015-04-16 at 6 20 58 pm

### CODE REVIEW

File: client/app/styles/_login.scss [hunk 1]
```diff
@@ -8,6 +8,7 @@
   top: 19%;
   left: 35%;
   border: 1px solid black;
+  direction: ltr;
```
Comment (amitaibu) [inline client/app/styles/_login.scss]: lets also change the plceholder to be in hebrew.

Comment (ceoaliongroo) [inline client/app/styles/_login.scss]: done.

File: behat/features/bootstrap/FeatureContext.php [hunk 2]
```diff
@@ -37,7 +37,7 @@ protected function loginUser($name, $password, $check_success = TRUE) {
     $element = $this->getSession()->getPage();
     $element->fillField('username', $name);
     $element->fillField('password', $password);
-    $submit = $element->findButton('Log in');
+    $submit = $element->findButton('התחברו');
```
Comment (amitaibu) [inline behat/features/bootstrap/FeatureContext.php]: ``התחבר``

**Comment by ceoaliongroo** [comment]
Merged.

**Pull request closed without merging**